includes/misc-functions.php

```diff
@@ -2113,3 +2113,27 @@ function give_get_user_agent() {
 	return $user_agent;
 
 }
+
+/**
+ * Return content by processing the shortcode and strip slashes
+ *
+ * @since 2.0.5
+ *
+ * @param string $content Content
+ *
+ * @return string $content Content
+ */
+function give_get_the_content( $content ) {
+	$content = do_shortcode( wpautop( stripslashes( $content ) ) );
```

Comment:
@raftaar1191 use `apply_filters( 'the_content', $content )` instead of `do_shortcode( wpautop( stripslashes( $content ) ) )`